src/Core/TileMesh.js

```diff
@@ -169,7 +169,11 @@ TileMesh.prototype.center = function center() {
 };
 
 TileMesh.prototype.OBB = function OBB() {
-    return this.geometry.OBB;
+    var zFactor = this.material.uniforms.zFactor.value;
+    var OBB = this.geometry.OBB;
+        OBB.box3D.min.z *= zFactor;
```

Comment:
Indentation is wrong here.

Reply:
don't understant! We need it to adapt Bbox to geoid tiles'bbox, zFactor is set by default to 1 so it will not have an impact on  DTM

Reply:
The [code indentation](https://en.wikipedia.org/wiki/Indentation_style) is wrong.

Btw, the code is wrong too since you're going to modify `OBB.box3D.min.z` (and max) each time the  ̀OBB()` function is called.
So for instance if `min.z = 1.0` and `zFactor = 3.0` initially, `min.z` will be: `3` then `9` then `27`...

Reply:
is there a posibility that in TileMesh i can have a notification when zFactor is changed ?

Reply:
@peppsac is there a copy function of OBB? what do you think if i make a copy of OOB and modify it?

Reply:
IMHO, I don't think you want to build a copy of the OBB.
You should be able to use the OBB.scale attribute to store the `zFactor` (ie: `scale = (1, 1, zFactor)`).

Reply:
Thank @peppsac for help, i tried it before but it did not work (i have holes as you can see in image), i think i need to modify OBB max min to adapt bbox for SSE.

![image](https://user-images.githubusercontent.com/30267875/32165962-b3306466-bd64-11e7-8433-5cf495514d21.png)

Reply:
My reasonning is the following: `OBB` is an `Object3D`. If you can't get `Object3D` properties (like scale) to work then it's a bug in `OBB`. 
This bug needs to be fixed (instead of adding another function to `OBB`).

Reply:
No, there is no bug and i understand what you said but I tried the scale and it did not work, because the problem is Box3D in OBB. Do not update it, the calculation of SSE will be wrong and the results as you can see in the picture above.

Btw, i do not think the code above is wrong, because in the case there is no geoid, zFactor is always equal to 1, there is no effect on MNT. In geoid case, when zFactor is greater than 1, i saw it runs just fine as you can see in picture below.

![image](https://user-images.githubusercontent.com/30267875/32214282-9f6f9a2c-be1e-11e7-81bb-314fd0a2c1a7.png)

Reply:
please dont take into account my last comment, i forgot update OBB, it work now by using OBB.scale and OBB.update .